modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -59,9 +61,33 @@
 			flex-basis: var( --n-tabs-heading-width );
 			flex-direction: var( --n-tabs-heading-direction );
 			flex-shrink: 0;
-			flex-wrap: wrap;
 			justify-content: var( --n-tabs-heading-justify-content );
 			gap: var( --n-tabs-title-gap );
+			overflow-x: var(--n-tabs-heading-overflow-x);
+			flex-wrap: var(--n-tabs-heading-wrap);
+			-ms-overflow-style: none;  /* IE and Edge */
+			scrollbar-width: none;  /* Firefox */
+
+			&::-webkit-scrollbar {
+				display: none; /* Hide scrollbar for Chrome, Safari and Opera */
+			}
+
+			&.e-scroll {
+				cursor: grabbing;
+				cursor: -webkit-grabbing;
+
+				&-active {
+					position: relative;
+
+					&::before {
+						content: '';
+						position: absolute;
+						inset-block: 0;
+						inset-inline: -1000vw;
```

Comment:
What is this about?
`inset-inline: -1000vw;`

Reply:
CC @hein-obox

Reply:
@ArmanPurtoian @louiswol94 I had added the `.e-scroll-active` to make sure that we couldn't hover over elements while dragging them to the left or right hand side.
I found that a bit annoying that the elements would receive a hover effect, while moving them. 

What's your take or approach about that?

Reply:
@louiswol94 

I had added the `.e-scroll-active` class, to be able to stop hover effects from happening when moving the elements from left to right.

I used an 'inset-inline: -1000vw` to make sure that this element would cover all tab elements.